library/tasks/HISAT2.wdl

```diff
@@ -304,3 +313,94 @@ task HISAT2InspectIndex {
     File log_file ="hisat2_inspect.log"
   }
 }
+
+task HISAT2RSEMSingleEnd {
+  File hisat2_ref
+  File fastq
+  String ref_name
+  String output_basename
+  String sample_name
+
+  # runtime values
+  String docker = "quay.io/humancellatlas/secondary-analysis-hisat2:v0.2.2-2-2.1.0"
+  Int machine_mem_mb = 15000
+  Int cpu = 4
+  Int disk = ceil((size(fastq, "GB")) * 100 + size(hisat2_ref, "GB") * 2 + 10)
+  Int preemptible = 5
+  Int max_retries = 0
+
+  meta {
+    description: "This HISAT2 alignment task will align paired-end fastq reads to transcriptome only. "
+  }
+
+  parameter_meta {
+    hisat2_ref: "HISAT2 reference"
+    fastq: "gz fastq file"
+    ref_name: "the basename of the index for the reference genome"
+    output_basename: "basename used for output files"
+    sample_name: "sample name of input"
+    docker: "(optional) the docker image containing the runtime environment for this task"
+    machine_mem_mb: "(optional) the amount of memory (MB) to provision for this task"
+    cpu: "(optional) the number of cpus to provision for this task"
+    disk: "(optional) the amount of disk space (GB) to provision for this task"
+    preemptible: "(optional) if non-zero, request a pre-emptible instance and allow for this number of preemptions before running the task on a non preemptible machine"
+    max_retries: "(optional) retry this number of times if task fails -- use with caution, see skylab README for details"
+  }
+
+  command {
+    set -e
+
+    # fix names if necessary.
+    if [ "${fastq}" != *.fastq.gz ]; then
+        FQ=${fastq}.fastq.gz
+        mv ${fastq} ${fastq}.fastq.gz
+    else
+        FQ=${fastq}
+    fi
+
+
+    tar --no-same-owner -xvf "${hisat2_ref}"
```

Comment:
Pretty sure `no-same-owner` is not needed here, was it required to make the WDL work?